server/lomas_server/dp_queries/dp_libraries/opendp.py

```diff
@@ -203,7 +203,7 @@ def get_output_measure(opendp_pipe: dp.Measurement) -> str:
     output_measure = opendp_pipe.output_measure
 
     if not isinstance(output_type, str):
-        if output_type.origin in ["SMDCurve", "Tuple"]:
+        if output_type.origin in ["SMDCurve", "Tuple"]:  # TODO constant.
```

Comment:
missing a TODO :)

Reply:
I just removed the const here. Probably not relevant to add constant since just used in this particular context.